Adds link to GH source and ECS kickstarter (#32)
playframework/akka-grpc-play-quickstart-scala: docs/src/main/paradox/index.md

For detailed documentation refer to https://www.playframework.com/documentation/latest/Home and https://developer.lightbend.com/docs/akka-grpc/current/ .

## Obtaining this example

You may download the code from [GitHub](https://github.com/playframework/play-scala-grpc-example) directly or you can 

> get kickstart you Play gRPC project on [Lightbend's Tech Hub](https://developer.lightbend.com/start/?group=play&project=play-scala-grpc-example). 

Reviewer: ```suggestion
kickstart your Play gRPC project on [Lightbend's Tech Hub](https://developer.lightbend.com/start/?group=play&project=play-scala-grpc-example). 
```
